src/functionalTest/resources/features/F-006/S-051.td.json

```diff
@@ -0,0 +1,166 @@
+{
+  "_guid_": "S-051",
+  "title": "must successfully create hearing role assignments for a user",
```

Comment:
this title must match with the scenario description like "must successfully create organisational role mapping for tribunal-caseworker and registrar"